tests/py/unit/test_tools_convert.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from ase import Atoms
+from ase.build import bulk
+from mat3ra.made.material import Material
+from mat3ra.made.tools.convert import from_ase, from_poscar, from_pymatgen, to_ase, to_poscar, to_pymatgen
+from pymatgen.core.structure import Element, Lattice, Structure
+
+PYMATGEN_LATTICE = Lattice.from_parameters(a=3.84, b=3.84, c=3.84, alpha=120, beta=90, gamma=60)
+PYMATGEN_STRUCTURE = Structure(PYMATGEN_LATTICE, ["Si", "Si"], [[0, 0, 0], [0.75, 0.5, 0.75]])
+
+POSCAR_CONTENT = """Si2
```

Comment:
As I understand, this is temporary, and in the future assets and fixtures will be moved to a separate location in standardized manner?

Reply:
Yes